src/views/quotations/SendQuotationForm.vue

```diff
@@ -0,0 +1,274 @@
+                separator=" "
+              ></b-form-tags>
+              <template #invalid-feedback>
+                You must provide at least 1 email recipient
+              </template>
+            </b-form-group>
+            <b-form-group
+              v-bind:label="$trans('Subject')"
+              label-for="offer_subject"
+              label-cols="3">
+              <b-form-input
+                autofocus
+                id="offer_subject"
+                size="sm"
+                v-model="offer.subject"
+                :state="isSubmitClicked ? !v$.offer.subject.$error : null"
+              ></b-form-input>
+              <b-form-invalid-feedback :state="isSubmitClicked ? !v$.offer.subject.$error : null">
+                {{ $trans("Please enter the email subject") }}
+              </b-form-invalid-feedback>
+            </b-form-group>
+            <b-form-group
+              label-cols="3"
+              v-bind:label="$trans('Body')"
+              label-for="offer_body"
+            >
+              <b-form-textarea
+                id="offer_body"
+                v-model="offer.body"
+                rows="3"
+              ></b-form-textarea>
+            </b-form-group>
+            <h6>{{ $trans("Attachments") }}</h6>
+            <p v-if="!documents.length">
+              {{ $trans("No attached documents to this quotation") }}
+            </p>
+            <p v-for="document in documents" :key="document.id">
+              {{ document.name }}
+              <b-button
+                class="btn button btn-danger quotation-pdf-button"
+                @click="downloadPdf"
+                v-if="document.is_pdf"
+                :disabled="loadingPdf"
+              >
+                <b-spinner small v-if="loadingPdf"></b-spinner>
+                {{ $trans('Preview quotation pdf') }}
+              </b-button>
+            </p>
+          </div>
+        </div>
+      </div>
+    </b-overlay>
+  </div>
+</template>
+<script>
+import { useVuelidate } from "@vuelidate/core";
+import { required } from "@vuelidate/validators";
+import { OfferService } from "@/models/quotations/offer.js";
+import {QuotationService} from '@/models/quotations/Quotation'
+
+
+export default {
+  setup() {
+    return { v$: useVuelidate() };
+  },
+  props: {
+    pk: {
+      type: [String, Number],
+      default: null
+    }
+  },
+  computed: {
+    isCreate() {
+      return !this.offer.id;
+    }
+  },
+  validations() {
+    return {
+      offer: {
+        subject: {
+          required
+        }
+      }
+    };
+  },
+  async created() {
+    await this.loadDocuments()
+    await this.loadData();
+  },
+  data() {
+    return {
+      isEdit: false,
+      isLoading: false,
+      loadingPdf: false,
+      isSubmitClicked: false,
+      quotationService: new QuotationService(),
+      offerService: new OfferService(),
+      recipients: [],
+      offer: {
+        quotation: this.$route.query.quotationId
```

Comment:
I think this is very ugly, and why not create an Offer model for this (you created one) and do this inside the `create()`